kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java

```diff
@@ -51,6 +52,7 @@ public static void afterAll() {
 
     @SneakyThrows
     @DisplayName("should create AVRO schema")
+    @Suite(suiteId = 11, title = "Schema Registry")
```

Comment:
for the future reference, if we're using the same suite id and title within same class, we can extract the values as constants

Reply:
done for SchemasTests and ConnectorsTests